packages/blockchain-wallet-v4-frontend/src/modals/Onfido/index.js

```diff
@@ -72,7 +74,7 @@ class OnfidoContainer extends React.PureComponent {
               Loading: () => <Loading />,
               NotAsked: () => (
                 <OnfidoIframe
-                  src={`${helperDomain}/wallet-helper/onfido?token=${sdkKey}`}
+                  src={`${helperDomain}/wallet-helper/onfido/#/token/${sdkKey}/docs/${docs}`}
```

Comment:
I'd rather this be a set of params with `?` and `&`s for easier recombination, more as a general rule

Reply:
queryString for instance, actually traditional query is parsed just by doing `query.split('?')[1].split('&').map((arg) => arg.split(','))`

Reply:
ah no, you also have to split jey and value with `.split('=')`